release/train_tests/e2e/test_persistence.py

```diff
@@ -47,6 +49,13 @@
     _resume_from_checkpoint,
 )
 
+# Add a unique ID to the storage path to avoid collisions between release test runs.
+TEST_ID = uuid.uuid4().hex[:4] + "_" + datetime.today().strftime("%Y-%m-%d_%H-%M-%S")
```

Comment:
As a follow-up, wondering if there is an environment variable that contains a unique ID for the release test that we could fetch here.

Reply:
I couldn't find any, cc: @ray-project/ray-ci